src/Castle.Core.Tests.WeakNamed/Castle.Core.Tests.WeakNamed.csproj

```diff
@@ -23,18 +23,11 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
-		<PackageReference Include="NUnit.Console" Version="3.11.1" />
-		<PackageReference Include="NUnit3TestAdapter" Version="3.16.1" />
-		<PackageReference Include="NUnitLite" Version="3.13.3" />
```

Comment:
I am not sure whether I understand & agree with the removal of these package references. This is a test project, no? So shouldn't it explicitly reference the packages necessary to run its tests?

(I am assuming that test execution still works after removing these is that they're indirectly referenced via the `Castle.Core.Tests.csproj` project reference. Or am I missing something else?)